docker/test/stateless/run.sh

```diff
@@ -92,6 +108,30 @@ if [[ -n "$USE_DATABASE_REPLICATED" ]] && [[ "$USE_DATABASE_REPLICATED" -eq 1 ]]
     MAX_RUN_TIME=$((MAX_RUN_TIME != 0 ? MAX_RUN_TIME : 9000))    # set to 2.5 hours if 0 (unlimited)
 fi
 
+
+# Wait for the server to start
+while true
```

Comment:
there should be a condition to break the loop. Like `for i in {1..100}`. And if nothing starts, the whole script should fail